src/DssSpell.sol

```diff
@@ -52,22 +52,140 @@ contract DssSpellAction is DssAction {
+            bidDuration: 6 hours,
+            auctionDuration: 6 hours,
+            liquidationRatio: 12500
+        });
+        addNewCollateral(UNIV2WBTCDAI_A);
+        DssExecLib.setChangelogAddress("UNIV2WBTCDAI",             UNIV2WBTCDAI_GEM);
+        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2WBTCDAI_A",  UNIV2WBTCDAI_JOIN);
+        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2WBTCDAI_A",  UNIV2WBTCDAI_FLIP);
+        DssExecLib.setChangelogAddress("PIP_UNIV2WBTCDAI",         UNIV2WBTCDAI_PIP);
+
+        // Onboard UNIV2AAVEETH-A
+        DssExecLib.addReaderToMedianWhitelist(
+            LPOracle(UNIV2AAVEETH_PIP).orb0(),
+            UNIV2AAVEETH_PIP
+        );
+        DssExecLib.addReaderToMedianWhitelist(
+            LPOracle(UNIV2AAVEETH_PIP).orb1(),
+            UNIV2AAVEETH_PIP
+        );
+        CollateralOpts memory UNIV2AAVEETH_A = CollateralOpts({
+            ilk: "UNIV2AAVEETH-A",
+            gem: UNIV2AAVEETH_GEM,
+            join: UNIV2AAVEETH_JOIN,
+            flip: UNIV2AAVEETH_FLIP,
+            pip: UNIV2AAVEETH_PIP,
+            isLiquidatable: true,
+            isOSM: true,
+            whitelistOSM: false,
+            ilkDebtCeiling: 3 * MILLION,
+            minVaultAmount: 2000,
+            maxLiquidationAmount: 50000,
+            liquidationPenalty: 1300,
+            ilkStabilityFee: FOUR_PCT,
+            bidIncrease: 300,
+            bidDuration: 6 hours,
+            auctionDuration: 6 hours,
+            liquidationRatio: 16500
+        });
+        addNewCollateral(UNIV2AAVEETH_A);
+        DssExecLib.setChangelogAddress("UNIV2AAVEETH",             UNIV2AAVEETH_GEM);
+        DssExecLib.setChangelogAddress("MCD_JOIN_UNIV2AAVEETH_A",  UNIV2AAVEETH_JOIN);
+        DssExecLib.setChangelogAddress("MCD_FLIP_UNIV2AAVEETH_A",  UNIV2AAVEETH_FLIP);
+        DssExecLib.setChangelogAddress("PIP_UNIV2AAVEETH",         UNIV2AAVEETH_PIP);
+
+        // Dai Savings Rate Adjustment
+        PotAbstract(DssExecLib.pot()).drip();
+        DssExecLib.setDSR(ONE_HUNDREDTH_PCT);
+
+        // Remove Permissions for Liquidations Circuit Breaker
+        address flipperMom = DssExecLib.flipperMom();
+        DssExecLib.deauthorize(DssExecLib.flip("PSM-USDC-A"), flipperMom);
+        DssExecLib.deauthorize(DssExecLib.flip("UNIV2DAIUSDC-A"), flipperMom);
+
+        // Fix for Line != sum lines rounding error issue (0.602857457497899800874246318932698818152722680 DAI)
+        VatAbstract vat = VatAbstract(DssExecLib.vat());
+        vat.file("Line", vat.Line() + 602857457497899800874246318932698818152722680);
+
+        // Add medianizers to the chain log
+        DssExecLib.setChangelogAddress("MED_ETH", OsmAbstract(DssExecLib.getChangelogAddress("PIP_ETH")).src());
+        DssExecLib.setChangelogAddress("MED_BAT", OsmAbstract(DssExecLib.getChangelogAddress("PIP_BAT")).src());
+        DssExecLib.setChangelogAddress("MED_WBTC", OsmAbstract(DssExecLib.getChangelogAddress("PIP_WBTC")).src());
```

Comment:
`MED_WBTC` is the same than `MED_RENBTC`. I think the key here should be just one called: `MED_BTC`. What do you guys think?

Reply:
yes

Reply:
I thought about this, but wanted to mirror how it's being done with the PIP. There is a PIP for both WBTC and RENBTC. It's also possible we switch one of these out in the future if liquidity on the actual tokens picks up. Makes sense to me to just duplicate it for each.

Reply:
So I think we've always known this as "the BTC feed", and we've chosen to use it as the src for the oracles. This breaks our convention, but maybe it's more useful this way. Paging @NiklasKunkel for input.

Reply:
Yeah I'm not sure what is the best here as there isn't `MED` for every collateral either. That is why I felt a bit hesitant to add the medianizers to the log on our call the other day.
So far the medianizer is a BTC price, not WBTC nor RENBTC.
I see the point that you want to make sure to get the correct value of a medianizer used by a certain `PIP`, however that can be achieved by doing `OsmAbstract(PIP_*).src()`. Which then makes me to question, why do we need the medianizers in the log at all?

Reply:
> there isn't MED for every collateral either.

This is why it's important to track them here. These are independent contracts in the Maker ecosystem that have authority granted to the pause proxy. I think we should view them as something external to collateral types, these are effectively Maker's oracles, more akin to our ticker, where the `PIP` is something related to an ILK.

I think oracles should be tracked, but they don't necessarily need to be correlated to an ilk.

Reply:
Yeah, ok, then in that case should be `MED_BTC`.

Btw the `PIP`s are not strictly related to an `ilk` either, as `ETH-A` and `ETH-B` use the same `PIP`, but yeah the `MED` is even in an extra layer of non-correlation.

Reply:
I'd almost rather call it `FEED_BTC` but we should probably stick with `MED` for technical clarity.

Reply:
if we call them `FEED_*` we could also add the `DSValue`s. So that is something to consider.
I feel this needs a bit further discussion, that's why I'd suggest leaving this out for this spell, but up to you guys if this will be added here and how it will be.

Reply:
Yeah I'm fine punting on this and having a discussion with Nik on the best way to track oracles.

Reply:
You guys want to remove all the `MED_*` from this spell?

Reply:
I agree I think it should MED_BTC. We may switch out the PIP sources (MED) later but for now the MED_BTC is just aggregating BTC prices and is being used as a Bitcoin Oracle by customers.